packages/smart-wallet/test/test-addAsset.js

```diff
@@ -1,12 +1,22 @@
 // @ts-check
 import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
-import { E } from '@endo/far';
+import { E, Far } from '@endo/far';
 import { buildRootObject as buildBankVatRoot } from '@agoric/vats/src/vat-bank.js';
-import { makeIssuerKit } from '@agoric/ertp';
+import { AmountMath, makeIssuerKit } from '@agoric/ertp';
 import { eventLoopIteration } from '@agoric/internal/src/testing-utils.js';
-import { makeScalarMapStore } from '@agoric/store';
+import { makeCopyBag, makeScalarMapStore } from '@agoric/store';
+import { makePromiseKit } from '@endo/promise-kit';
+import bundleSource from '@endo/bundle-source';
+import { makeMarshal } from '@endo/marshal';
+import { resolve as importMetaResolve } from 'import-meta-resolve';
 import { makeDefaultTestContext } from './contexts.js';
-import { makeMockTestSpace } from './supports.js';
+import { ActionType, headValue, makeMockTestSpace } from './supports.js';
+import { makeImportContext } from '../src/marshal-contexts.js';
+
+const { Fail } = assert;
+
+const importSpec = spec =>
+  importMetaResolve(spec, import.meta.url).then(u => new URL(u).pathname);
```

Comment:
I think it would be nice to avoid taking on an `import-meta-resolve` dependency just for this.

Reply:
I'm not aware of an alternative. It's pretty normal for tests that use `bundleSource`, no?

In master, I think the dependency was/is already there.